bin/resin-preload

```diff
@@ -132,6 +131,12 @@ buildOutputStream.pipe(process.stdout)
 
 streamToPromise(buildOutputStream)
 .then(() => {
+  if (options.apiToken) {
+    return resin.token.set(options.apiToken)
```

Comment:
This will work I think, but will also set your user-wide token, updating the token for the CLI, which might cause some unexpected behaviour for some users, and is an odd side effect.

You can fix that by specifying a different data directory when you create the SDK, by passing `dataDirectory: <tmpDir>` in `resinSdk({ })` above. You'll need some library to give you a new tmp dir each time though, and to clean it up after too.

Is the behaviour where an existing CLI token is used automatically desirable? If it's not, you can just always use a temp dir. If it is, though you'll want to use a temp dir only if `options.apiToken` is provided, and stick with the default otherwise (so any existing tokens there get found).